pkg/metaserver/util/util.go

```diff
@@ -66,7 +66,7 @@ func UnsafeResourceToKind(r string) string {
 	switch {
 	case strings.HasSuffix(k, "ies"):
 		return strings.TrimSuffix(k, "ies") + "y"
-	case strings.HasSuffix(k, "es"):
+	case strings.HasSuffix(k, "ses"):
```

Comment:
Is it need for case `es`?

Reply:
In the implementation of k8s, `es` is added only when the word ends with `s`. https://github.com/kubernetes/apimachinery/blob/master/pkg/api/meta/restmapper.go#L142